docs/userGuide/components/panel.md

```diff
@@ -64,13 +64,32 @@
 
 <tip-box border-left-color="#00B0F0">
   <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
-  <code>primary</code> type<br>
-  <panel header="#### primary type panel" type="primary">
+  <p>Click the Panels to see the expanded style.<p>
```

Comment:
Er... the commit `Docs: Update panel section` doesn't seem to be related to this PR? :P

Reply:
As there were new types introduced, this section has to be updated. I did some extra formatting to make it more explicit in presenting the different styles.